userspace/falco/outputs_grpc.cpp

```diff
@@ -64,5 +64,9 @@ void falco::outputs::output_grpc::output(const message *msg)
 	auto host = grpc_res.mutable_hostname();
 	*host = m_hostname;
 
+	// tags
+	auto tags = grpc_res.mutable_tags();
+	*tags = {msg->tags.begin(), msg->tags.end()};
```

Comment:
Let's check what we end up with when a Falco rule does **not** have tags defined.

Do we obtain `...,"tags":[],...` or not? I'd prefer a clean JSON, but that's my personal preference only :)

Reply:
I don't have any preferences. Anyway, I agree with @leodido that a test covering such a scenario (especially for the JSON output format) would be really helpful to clarify the expected result.

Reply:
Thank you for your review! I opened a PR to address these concerns, so we can eventually move this conversation there: https://github.com/falcosecurity/falco/pull/1733